zproject/backends.py

```diff
@@ -1224,6 +1225,8 @@ def social_auth_finish(backend: Any,
         return HttpResponseRedirect(reverse('zerver.views.registration.find_account'))
 
     if inactive_user:
+        logging.info("Failed attempt to login from an inactive account with ID: %s",
+                     return_data['inactive_user_id'])
```

Comment:
Can we easily do `"Failed login attempt for deactivated account: %s@%s" % (inactivae_user_id, realm.string_id)?  That would be consistent with other logging of a user object.

Reply:
I've added another key `realm_string_id` to `return_data` instead of getting realm object before we actually need it in `social_auth_finish`.